pantry-finder-api/spec/serializers/event_serializer_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+describe EventSerializer do
+  it 'JSON Event Display fields' do
+    described_class.new(
+      Event.new(event_name: 'big event', address1: 'addr 1',
+                city: 'the town', state: 'OH', zip: '12345',
+                pt_latitude: '40.1479133',
+                pt_longitude: '-83.0959877', loc_id: '6')
+    )
```

Comment:
There's no assertion in this test.

Reply:
I couldn't match on service.  I will try again it again and maybe we can figure it out.

Reply:
I left it in as is because I actually found a problem by running it as is.